### DISCUSSION TITLE
2.31: flood of eof messages

### DISCUSSION THREAD

**Discussion by tu-maurice** [seaweedfs/seaweedfs#1925] [Q&A]
Hi,

I'm trying to setup a seaweedfs cluster on three servers. To have no SPOF I'm trying to set up a master on each of the servers right away using the 2.31_large_disk docker image. Running one master has been successful so far but once I specify the -peers parameter with the other two masters, I get the following message in the log every 20 seconds:

From server1: `masterclient.go:120] master masterClient failed to receive from server1:9333: EOF`

Same thing for the other master each failing to receive from themselves.

I'm starting my server1 master as follows: `weed master -ip server1 -peers server2:9333,server3:9333`

**Comment by tu-maurice** [comment]
My bad. it went away in 2.34.

**Reply by chrislusf** [reply]
Good that it went away, but there should be no master related changes between 2.31 and 2.34.

**Reply by tu-maurice** [reply]
:man_shrugging:
